portfolio/src/main/webapp/index.html

```diff
@@ -9,22 +9,23 @@
   <body>
     <div id="content">
       <ul id="navbar">
-        <li class="nav-button current" ><a>Home</a></li>
-        <li class="nav-button"><a href="introduction.html">Introduction</a></li>
-        <li class="nav-button"><a>Projects</a></li>
-        <li class="nav-button"><a>Gallery</a></li>
+        <li class="nav-button current" ><a class="nav-link">Home</a></li>
+        <li class="nav-button"><a href="introduction.html" class="nav-link">Introduction</a></li>
+        <li class="nav-button"><a href="projects.html" class="nav-link">Projects</a></li>
+        <li class="nav-button"><a href="gallery.html" class="nav-link">Gallery</a></li>
         <li>
           <a class="ext-link" href="https://github.com/Lorraine-Lyu"><img alt="link to github" src="images/github.svg"></a>
         </li>
         <li>
           <a class="ext-link" href="https://github.com/Lorraine-Lyu"><img alt="link to linkedin" src="images/linkedin.svg"></a>
```

Comment:
80 character lines.

Reply:
Just saw your comment above about line length and breaking out the navbar. Either change to 80 character lines now or create a bug and add a TODO.